app.js

```diff
@@ -31,8 +38,9 @@ if (isRateLimiterEnabled) {
       await rateLimiter.consume(ctx.ip);
       await next();
     } catch (rejRes) {
+      const errorSpecialty = rejRes.message || JSON.stringify({ ...rejRes, ip: ctx.ip });
       ctx.status = 429;
-      ctx.body = 'Too Many Requests';
+      ctx.body = `Too Many Requests for ${ctx.ip}. Specialty: ${errorSpecialty}`;
```

Comment:
`. Specialty: ${errorSpecialty}` should not be in the responce body. It may contain internal information